src/components/Dashboard/NextToJumpBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import moment from "moment";
+import { NextToJump } from "../../types/meets";
+import { Loader } from "../Loader";
+
+type Props = {
+  NextToJump?: NextToJump[];
+};
+
+export const NextToJumpBanner: React.FC<Props> = ({ NextToJump }) => (
```

Comment:
please add this to the `index.ts` file in the folder so that it can be imported with the other dashboard components